src/pages/GamePage/components/CharacterFigure.js

```diff
@@ -0,0 +1,28 @@
+import React from 'react';
+import {
+  halfOrc, elf, dwarf, human,
+} from '../../../assets';
+
+export default function CharacterFigure({ race }) {
+  let figure = null;
+  switch (race) {
```

Comment:
Switch structure should be a function itself. Render should only be resposible for renderinmg stuff, extra logic should be placed in functions